### ISSUE TITLE
Fix display of soundtrack and subtitle language on Samsung TV App

### ISSUE THREAD

**Issue by subru** [Stremio/stremio-features#877]
**Stremio Version**
Stremio Version 1.3.0 on Tizen OS 8.0 firmware 1411

**Is your feature request related to a problem? Please describe.**
The Samsung TV App in general works very nicely but unfortunately both the soundtrack and subtitle language shows up as unknown so I have to brute force try each option before I get the right language.

**Describe the solution you'd like**
I would like soundtrack and subtitle language metadata to be displayed correctly, i.e. English, Spanish, Hindi etc instead of unknown.

**Describe alternatives you've considered**
When I play on firestick, the soundtrack and subtitle language metadata is displayed correctly.

**Comment by MrMonkeyz** [comment]
i agree with this one as it is annoying when all you see is a load of unknowns.. yet it can be seen fine if use android and selected correctly

i even find if you set the option of subtitles to none in the app options it will still show them regardless

**Comment by MrMonkeyz** [comment]
Let's hope this gets fixed

**Comment by midozalouk** [comment]
@MrMonkeyz 
Hello Sir, you were developing a version of Stremio for Tizen 4 , has it gone?!

**Comment by MrMonkeyz** [comment]
It wasn't me unfortunately as trying to find out who the user was.. as surely someone knows how to make it work on tizen 4 even if stremio can't be bothered...  I can see why people are using the alternative app to stremio
